src/python/WMCore/WMSpec/WMTask.py

```diff
@@ -926,10 +927,19 @@ def setSubscriptionInformation(self, custodialSites = None, nonCustodialSites =
             outputDataset = entry["outputDataset"]
             outputModule = entry["outputModule"]
 
-            primDs = outputDataset.split('/')[1]
-            tier = outputDataset.split('/')[3]
+            dsSplit = outputDataset.split('/')
+
+            primDs = dsSplit[1]
+
+            procDsSplit = dsSplit[2].split('-')
+            skim = ( len(procDsSplit) == 4 )
```

Comment:
Yes, only Tier0 should ever need to use the two new parameters. Everyone else will get the defaults. Only Tier0 workflows have these specific requirements about putting one type of output in one place and another type of output in another place.

Tier0 processed dataset names are constructed using standard WMCore code in StdBase. I would assume any kind of lexicon check is done there.